boa/explorer.py

```diff
@@ -1,10 +1,21 @@
-import json
 import time
 from typing import Optional
 
-import requests
+from boa.rpc import json
 
-SESSION = requests.Session()
+try:
+    from requests_cache import CachedSession
+
+    SESSION = CachedSession(
```

Comment:
the CachedSession API is really quite interesting https://requests-cache.readthedocs.io/en/stable/modules/requests_cache.session.html#requests_cache.session.CachedSession

i suggest we add:
```
expire_after=3600*24*7,
stale_if_error=True,
stale_while_revalidate=True,
```
```

Reply:
actually since stale_while_revalidate removes latency, we can actually set expire_after to a much tigher value, like 1 day or 1 hour.